### ISSUE TITLE
Stop upgrading to latest WebUI versions

### ISSUE THREAD

**Issue by digital-soul18** [TheLastBen/fast-stable-diffusion#1897]
Since Automatic1111 is so unstable I'd like to request that we bring back the option for users to NOT upgrade to latest version. (ie. more specifically we choose when to upgrade our instances) which the older versions of this repo used to do. Right now we force reset to the latest WebUI and it is frustrating how many new things break with every new release.

**Comment by Ilija712** [comment]
When you say automatic1111 is unstable do u mean elements become unresponsive all the time cuz thats happening to me too and its so frustrating never experienced this before. Sometimes generate button stops working, sometimes save button, sometimes highres fix checkbox and so on. Are you also having these problems?

**Comment by digital-soul18** [comment]
Yes. Plus img2img now doesn't work. Clicking on a image to show in lightbox
also doesn't work. Just tired of these bugs. Clearly there is no unit tests
in webui so that's why these bugs get through.

**Comment by dizzyjung** [comment]
I fought the error for 2 days, but it was a meaningful update for me. I deleted the existing installation and installed a new one. Several extensions were also updated, and above all, it works more comfortably and stably. 2.5x upscale works well

**Comment by digital-soul18** [comment]
I just want the option in theLastBen automatic 1111 to not force to
download the latest commit and instead allow a tickbox to upgrade

**Comment by Topzie** [comment]
Same.

**Comment by TheLastBen** [comment]
@digital-soul18 I'm modifying some files from the A1111 repo, and they are downloaded directly from the latest repo, so stopping the git pull will cause issues

**Comment by digital-soul18** [comment]
@TheLastBen but can't we just install once all dependencies on Google Drive for a first time setup on G drive and then we can launch just that instance without having to git pull the latest version? I did achieve this in previous versions of the repo but since the python upgrade in Collab its killed that old version of collab. Would really appreciate a way to keep our environments stable from the chaotic auto1111 repo.
